scripts/skill_auditor/metrics_extractor.py

```
@@ -0,0 +1,88 @@
+        description = simple_match.group(1).strip() if simple_match else ""
+
+    # Extract quoted phrases (deterministic regex)
+    quoted_phrases = re.findall(r'"([^"]+)"', description)
+
+    # Extract domain indicators (exact regex from v5 agent)
+    domain_pattern = (
+        r'\b(SKILL\.md|\.skill|YAML|Claude Code|Anthropic|'
+        r'skill|research|validation|compliance|specification|frontmatter)\b'
+    )
+    domain_matches = re.findall(domain_pattern, description, re.IGNORECASE)
+    domain_indicators = list(set(match.lower() for match in domain_matches))  # Unique, case-normalized
```

Ruff (0.14.5)

Unnecessary generator (rewrite as a set comprehension)

Rewrite as a set comprehension

(C401)